codersdk/deployment.go

```diff
@@ -1580,3 +1644,25 @@ type DeploymentStats struct {
 	Workspaces   WorkspaceDeploymentStats    `json:"workspaces"`
 	SessionCount SessionCountDeploymentStats `json:"session_count"`
 }
+
+type CLISSHConfigResponse struct {
```

Comment:
I think this could just be `SSHConfigResponse`, because it's used for more than the CLI.

Reply:
True